From a34d690d17765b9b7d26bbb7587dd5535e709082 Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Mon, 5 Aug 2024 13:22:17 +0800
Subject: [PATCH 102/109] WIN2030-15279:fix(bmc2.5):Bootsel IO pins was changed
 by mistake

Changelogs:
1.The bootsel pins was changed to input mode by mistake in som_reset_control(),
  in other words, the bootsel was controlled by hardware dip switch.
  This is not expected behavious when the setting stored in the eeprom is actually
  in software controll bootsel mode.
  Deleted the set_bootsel() call in som_reset_control().

Signed-off-by: linmin <linmin@eswincomputing.com>
Change-Id: I4f04e1ff78c8f6b3381059d2e141cc245ad99d6a
---
 Core/Src/hf_power_process.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Core/Src/hf_power_process.c b/Core/Src/hf_power_process.c
index 3e38651..3348c7a 100644
--- a/Core/Src/hf_power_process.c
+++ b/Core/Src/hf_power_process.c
@@ -217,7 +217,6 @@ void som_reset_control(uint8_t reset)
 		uart_deinit(UART4);
 		uart_deinit(USART6);
 		SPI2_MASTER_CS_LOW();
-		set_bootsel(0, 0);
 	} else {
 	// 	GPIO_InitStruct.Pin = BOOT_SEL0_Pin | BOOT_SEL1_Pin | BOOT_SEL2_Pin | BOOT_SEL3_Pin;
 	// 	GPIO_InitStruct.Mode = GPIO_MODE_INPUT;
-- 
2.25.1

